css/modules.css

```diff
@@ -145,15 +153,9 @@ input[type="checkbox"]:checked {
   font-family: 'Oswald', sans-serif;
 }
 
-.share:hover {
-  background-color: #990000;
-  color: #D1CFD5;
-}
-
-
 /****************** Footer *****************/
 
-.footer{
+footer {
   margin-top: 42px;
   padding-top: 25px;
   font-family: 'Droid Serif', serif;
```

Comment:
The lack of this being a base property set to the body is making you have to repeat this line. If it is set to the body, all text will inherit the font-family of Droid-Serif. Specific elements can have a different font-family applied to them a needed (i.e. The main logo heading uses Yellowtail). This prevents repetition of properties in your css.